test/VectorInstructions/float_vector_constant_llvm34.ll

```diff
@@ -0,0 +1,244 @@
+; FIXME: Need to upstream floating point support to make this test pass
```

Comment:
I would remove this test from this PR and add it when floating-point support is upstreamed.